Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -185,9 +185,13 @@ class TabTrayController: UIViewController, UITabBarDelegate, UITableViewDelegate
         let signinItem = UIBarButtonItem(title: NSLocalizedString("Sign in", comment: "Button that leads to Sign in section of the Settings sheet."),
             style: .Plain, target: self, action: "SELdidClickDone")
         signinItem.enabled = false
-        // TODO: Vertically center the add button.  Right now, it's too high in the containing bar.
+
         let addTabItem = UIBarButtonItem(barButtonSystemItem: .Add, target: self, action: "SELdidClickAddTab")
+        
+        let topInset:CGFloat = 10.0;
```

Comment:
nit: space before `CGFloat`.  How are you calculating 10?  Is this from trial and error?  If so, can we try to figure out a way to deduce this from the toolbar and image height?